__tests__/data/persistiq.json

```diff
@@ -1,13 +1,12 @@
 [
   {
-    "description": "[::IDENTIFY::]-> Update call with leadId in externalId",
+    "description": "[::IDENTIFY::]-> Update call with leadId in externalId and no email",
```

Comment:
Can we add the company field in the test case too?

Reply:
yeah sure